src/RPC.js

```diff
@@ -418,7 +418,7 @@ class RPC {
             }
             let attempts, timeout = false;
             let resp;
-            while (true) {
+            while (constant.ReqCondition) {
```

Comment:
`constant.ReqCondition` here addresses the rule https://eslint.org/docs/rules/no-constant-condition